app/components/content/quote_list_component.rb

```diff
@@ -0,0 +1,26 @@
+module Content
+  class QuoteListComponent < ViewComponent::Base
+    attr_reader :quotes
+
+    def initialize(quotes)
+      super
+
+      @quotes = quotes
+    end
+
+  private
+
+    def before_render
+      validate!
+    end
+
+    def validate!
+      quotes.each_with_index do |quote, idx|
+        %i[heading text accreditation].each do |required_key|
```

Comment:
Nitpick: I would have the array of symbols as a constant:
``` Ruby
REQUIRED_KEYS = %i[heading text accreditation].freeze
```
With the constant being defined at the top of the module. Makes it a bit more readable. But that's just my preference and please feel free to ignore me -:)

Reply:
Yeah agreed, that is cleaner - pushed an update